docs/user/07-70-function-specification.md

```diff
@@ -0,0 +1,254 @@
+Apart from these two default codes, you can define custom responses in all Node.js runtimes using the **event.extensions.response** object.
+
+This object is created by the Express framework and can be customized. For more information, read [Node.js API documentation](https://nodejs.org/docs/latest-v12.x/api/http.html#http_class_http_serverresponse).
+
+This example shows how to set such a custom response in Node.js for the HTTP status code `400`:
+
+```js
+module.exports = {    
+    main: function (event, context) {
+        if (event.extensions.request.query.id === undefined) {
+            res = event.extensions.response;
+            res.status(400)
+            return
+        }
+        return "42"
+    }
+}
+```
+
+</details>
+<details>
+<summary label="Python">
+Python
+</summary>
+
+By default, a failing Function simply throws an error to tell the Event Service to reinject the event at a later point. Such an HTTP-based Function returns the HTTP status code `500`. In case you manage to invoke a Function successfully, the system returns the default HTTP status code `200`.
+
+Apart from these two default codes, you can define custom responses in all Python runtimes using the **HTTPResponse** object available in Bottle.
+
+This object will need to be instantiated and can be customized. For more information, read [Bottle API documentation](https://bottlepy.org/docs/dev/api.html#the-response-object).
+
+The following example shows how to set such a custom response in Python for the HTTP status code `400`:
+
+```python
+from bottle import HTTPResponse
+
+SUPPORTED_CONTENT_TYPES = ['application/json']
+
+def main(event, context):
+    request = event['extensions']['request']
+
+    response_content_type = 'application/json'
+    headers = {
+        'Content-Type': response_content_type
+    }
+
+    status = 202
+    response_payload = {'success': 'Message accepted.'}
+
+    if request.headers.get('Content-Type') not in SUPPORTED_CONTENT_TYPES:
+        status = 400
+        response_payload = json.dumps({'error': 'Invalid Content-Type.'})
+
+    return HTTPResponse(body=response_payload, status=status, headers=headers)
+```
+
+</details>
+</div>
+
+## Override runtime image
+
+You can use a custom runtime image to override the existing one. Your image must meet all the following requirements:
+
+- Expose the workload endpoint on the right port
+- Provide liveness and readiness check endpoints at `/healthz`
+- Fetch sources from the path under the `KUBELESS_INSTALL_VOLUME` environment
+- Security support. Kyma runtimes are secure by default. You only need to protect your images.
+
+> Note: For better understanding, you can look at [main dockerfiles](https://github.com/kyma-project/kyma/blob/main/resources/serverless/templates/runtimes.yaml) which are responsible for building the final image based on the `base_image` argument which you as a user can override and what we are doing in [this tutorial](/docs/user/03-110-override-runtime-image.md).
```

Comment:
```suggestion
> **NOTE:** For better understanding, you can look at the [main Docker files](https://github.com/kyma-project/kyma/blob/main/resources/serverless/templates/runtimes.yaml), which are responsible for building the final image based on the `base_image` argument, which you, as a user, can override and what we are doing in [this tutorial](/docs/user/03-110-override-runtime-image.md).
```

I think we should simplify this sentence and split it in two or three shorter ones.